core/transport/src/main/proto/proto/shared/acl.proto

```diff
@@ -50,3 +57,21 @@ message AccessControlList {
     optional bool isDefault = 7;
     optional bool isEmpty = 8;
 }
+
+// next available id: 10
+message NewAccessControlList {
```

Comment:
I find that the `New` keyword is not future proof. What if we need a new new version in the future? Is there a naming scheme that can represent this evolution better?

Reply:
v2